build.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Might want to remove this file and other build related files as I think they're related to the jenkins build